tensorflow/compiler/mlir/tensorflow/translate/tf_mlir_translate.cc

```diff
@@ -38,6 +38,7 @@ limitations under the License.
 #include "tensorflow/core/platform/errors.h"
 #include "tensorflow/core/platform/protobuf.h"
 #include "tensorflow/core/protobuf/graph_debug_info.pb.h"
+#include "tensorflow/core/util/tensor_bundle/byte_swap_tensor.h"
```

Comment:
the BUILD entry for this target needs to depend directly on the build target that exports the `byte_swap_tensor.h` header.

Reply:
Hi @cantonios ,

Thanks for the clarification! I've added target `//tensorflow/core/util/tensor_bundle:byteswaptensor` in the dep list of the target `translate_lib` in its BUILD entry. Could you please take a look?